trie/committer.go

```diff
@@ -142,12 +142,10 @@ func (c *committer) store(path []byte, n node) node {
 	// We have the hash already, estimate the RLP encoding-size of the node.
```

Comment:
This comment is stale now I think, we're not estimating the size anymore.
Also we could create a pool for rlp.NewEncodeBuffer within `nodeToBytes` to reduce allocs